runner/database_mongo.go

```diff
@@ -0,0 +1,51 @@
+package runner
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+func (ec EvalContext) evalMongo(panel *PanelInfo, dbInfo DatabaseConnectorInfoDatabase, server *ServerInfo, w *ResultWriter) error {
+	_, conn, err := ec.getConnectionString(dbInfo)
+	if err != nil {
+		return err
+	}
+
+	authDB, ok := dbInfo.Extra["authenticationDatabase"]
+	if !ok {
+		authDB = "admin"
+	}
+
+	// EJSON.stringify required to make it possible to process output
+	eval := fmt.Sprintf("'EJSON.stringify(%s)'", panel.Content)
+
+	args := []string{conn, "--quiet", "--authenticationDatabase", authDB, "--eval", eval}
+	stdoutStderr, err := exec.Command("mongosh", args...).CombinedOutput()
```

Comment:
mongo can't be mixing stdout and stderr. It should only be one of these. The other one will contain actual log lines. Most likely they only dump the JSON object on stdout not stderr.

Reply:
Should be better now